### PR TITLE
feat: markdown table links check

### PR DESCRIPTION
**Pull request by alexandrglaznyov**
This pull request includes the following changes:

- Moved JavaScript-related files (index.js, package.json, package-lock.json) to ./github/scripts directory.
- Added a new GitHub Action named update-service-status-badges.yml.
- Configured manual triggering of the GitHub Action using workflow_dispatch.
- Enhanced the functionality of the action by adding a cron schedule.
- Added the commit message to "docs: update url service status badges".
- Added a script that edits the README.md file to check link status based on the URL column in each service table
- Added a helper directory with corresponding code for link checking
- Updated the README.md file to separate the Services table into multiple tables (RPC, API, gRPC, etc.) for better visibility of individual service statuses
- Added a status column to the Block Explorers, Documentation / Guides, and Dashboards tables for monitoring link functionality

### CODE REVIEW

File: .github/workflows/update-service-status-badges.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - main
+    paths:
+      - '**/README.md'
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+    
+jobs:
+  run-script:
+    runs-on: ubuntu-latest
```
Comment (ccamel) [inline .github/workflows/update-service-status-badges.yml:14]:
```suggestion
    runs-on: ubuntu-22.04
```

File: README.md [hunk 2]
```diff
@@ -91,4 +158,4 @@ Please submit a pull request if you know any resources that might be helpful to
 Don't forget to check out OKP4 health files:
 
 - [Contributing](https://github.com/okp4/.github/blob/main/CONTRIBUTING.md)
-- [Code of conduct](https://github.com/okp4/.github/blob/main/CODE_OF_CONDUCT.md)
+- [Code of conduct](https://github.com/okp4/.github/blob/main/CODE_OF_CONDUCT.md)
```
Comment (ccamel) [inline README.md:161]: `MD047/single-trailing-newline`- Files should end with a single newline character

```suggestion
- [Code of conduct](https://github.com/okp4/.github/blob/main/CODE_OF_CONDUCT.md)

```

**Review by ccamel — CHANGES_REQUESTED** [review]
Looks awesome! :+1:  Just some minor changes requested on my side - it would so great it if you could take them into consideration.

File: .github/workflows/update-service-status-badges.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/README.md'
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
```
Comment (ccamel) [inline .github/workflows/update-service-status-badges.yml:11]: workflows/update-service-status-badges.yml:11:1: [error] trailing spaces (trailing-spaces)

```suggestion

```

**Review by ccamel — APPROVED** [review]
LGTM! Thanks for your great contribution.

**Review by amimart — APPROVED** [review]
Looks good! Thanks a lot 🙏

File: .github/scripts/package.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/alexandrglaznyov/okp4-README.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/alexandrglaznyov/okp4-README/issues"
+  },
+  "homepage": "https://github.com/alexandrglaznyov/okp4-README#readme",
```
Comment (bdeneux) [inline .github/scripts/package.json:18]: Shouldn't we put the correct repo url here? @ccamel  

```suggestion
    "url": "git+https://github.com/okp4/awesome.git"
  },
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/okp4/awesome/issues"
  },
  "homepage": "https://github.com/okp4/awesome#readme",
```

**Review by bdeneux — APPROVED** [review]
Very nice ! Thanks 👍

Just one question about `package.json ` but otherwise it seems very good ! Thanks.

**Comment by ccamel** [comment]
Everything looks good! Time to merge. :smile:

**Pull request merged**